js/startup/mouseForwardBack.js

```diff
@@ -0,0 +1,26 @@
+import { remote } from 'electron';
+
+// handles mouse back/forward buttons
+
+export default function () {
+
+  // app-command works on Windows only
+  $(window).on("app-command", (e, cmd) => {
```

Comment:
Can you please import jQuery in this file? It's available and made global because other files in the app are importing it, but we'd like each file to explicitly declare it's dependencies (better for portability and refactoring).